README.md

```diff
@@ -1098,6 +1099,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 
 - [Bitwarden](https://bitwarden.com/) `⚠` - Password manager with webapp, browser extension, and mobile app. ([Source Code](https://github.com/bitwarden/core)) `AGPL-3.0` `C#`
 - [bitwarden_rs](https://github.com/dani-garcia/bitwarden_rs) - Lightweight Bitwarden server API implementation written in Rust. `GPL-3.0` `Rust`
+- [KeePassXC](https://keepassxc.org/) -  A cross-platform community-driven KeePass client. ([Source Code](https://github.com/keepassxreboot/keepassxc)) `GPL-2.0` `C++`
```

Comment:
KeepassXC is very good but it's desktop software, we don't list these here.

Reply:
ah okay, I'll remove that.